### ISSUE TITLE
[Feature Request]: Highlight changed variables

### ISSUE THREAD

**Issue by noscript** [puremourning/vimspector#71]
It would be very useful to see which variables change between steps. For example lldb's TUI highlights those with red:

![variables](https://user-images.githubusercontent.com/717109/69503709-3e33a500-0f25-11ea-99c4-485a4e0baecc.gif)

**Comment by puremourning** [comment]
I think it’s possible. I Added a card to trello

**Comment by puremourning** [comment]
@noscript  I have pushed a branch that implements a little asterisk marker when a var changes in the locals/watches window.

https://github.com/puremourning/vimspector/tree/variables

let me know what you think. improvements around visuals like using highlighting can come later, but this is easy and quick. took longer than necessary because the code for this was really bad and I had to refactor it to make this vaguely sane. glad I did.
